pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -1143,3 +1143,30 @@ def tool_call_id(self) -> str:
 HandleResponseEvent = Annotated[
     Union[FunctionToolCallEvent, FunctionToolResultEvent], pydantic.Discriminator('event_kind')
 ]
+
+
+@dataclass(repr=False)
+class BaseCountTokensResponse:
```

Comment:
Do we need this entire response? Or could `count_tokens` just return the total tokens counted, or even better: a `Usage` object we can use with `Usage.incr`?